### DISCUSSION TITLE
Configure Table Issue

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [netbox-community/netbox#8215] [General]
Good Morning,

I am running Netbox 3.1.3.  I tried to edit the table that is displayed in the Contacts page using the Configure Table button.  I remove the Group from the table and now I get the error:
<class 'django.template.exceptions.TemplateDoesNotExist'>

utilities/templatetags/tag.html

Python version: 3.8.8
NetBox version: 3.1.3

Any help would be great!

**Comment by oneCrazyAdmin** [comment]
I am not sure what really happened, but I cleared out the data column in my specific user and it started working again.

**Reply by jeremystretch** [reply]
This is related to bug #8204, which has been fixed in v3.1.4.
